README.md

```diff
@@ -20,8 +27,23 @@ To start the container run:
 ```
 make
 ```
+> to see whats happening with this command see the `Makefile`.
+
+### Install dependencies
+```yarn install```
```

Comment:
The dockerfile is meant to handle this for you.. wonder why it's not doing it correctly. We could bake these steps in to the Dockerfile?

Reply:
Including the linking and yarn install on the example app.